container/centraldb/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM postgres:latest
```

Comment:
https://images.chainguard.dev/directory/image/postgres/overview könnte vielleicht ein gutes Replacement sein

Reply:
Das folgt in späteren Versionen. Ich hab Issue #2 hierzu erstellt.